### ISSUE TITLE
[Broken Website] Partial CSS is being applied causing the page to be unreadable.

### ISSUE THREAD

**Issue by VivekSharma1897** [darkreader/darkreader#6350]
# Broken Website Report

## Website Address
https://myanimelist.net/

## Steps To Reproduce
- Open the website in background tab.
- Let the page load completely.
- Switch to the loaded page.

## Expected Behavior
Background should be dark with visible text in white.

## Actual Behavior
Partial CSS is applied causing text to be white but background retains previous (white) CSS.

## Screenshots
![image](https://user-images.githubusercontent.com/37402553/126897739-f1212a9b-8a77-4533-899a-337d57ed7484.png)
![image](https://user-images.githubusercontent.com/37402553/126897751-54d81822-7db4-4417-92d1-ad0b763c8824.png)

## System Information
- OS: Windows 10 Version 21H1 (Build 19043.1110)
- Browser: 91.0.4472.164
- Dark Reader Version: 4.9.34

## Additional Context
Issue was logged earlier as well but closed without solution. Link: https://github.com/darkreader/darkreader/issues/2512

**Comment by Gusted** [comment]
Hi @VivekSharma1897!

1) Is this consistent behavior or happen once in a while?
2) Do you have any special settings related to myanimelist.net? That could conflict with Dark reader?

The issue was closed as it showed similar symptoms to another issue and was fixed in the next dark reader release.

**Comment by VivekSharma1897** [comment]
Hey @Gusted ,
1. Yes it is very consistent issue, happens to me all the time. 

2. I don't have any other settings enabled which may affect Dark Reader.

**Comment by Gusted** [comment]
Hmm, I've went back to the stock version 4.9.34 as I might been on a commit which already fixed the issue, but it still doesn't occur to me. Could you try to create a new profile?

![image](https://user-images.githubusercontent.com/25481501/126901437-0ad93181-c09f-4648-b89c-73899de009dd.png)

It might be due to some hidden setting that's enabled or something else.

**Comment by VivekSharma1897** [comment]
You are right. I just tried in New Profile and Incognito and I found uBlock Origins is causing the issue. After disabling uBlock Origins (either completely or just for MyAnimeList) the DarkReader worked properly. 
Is it possible to get a solution from DarkReader for this conflict or we will have to somehow work around it?

**Comment by Gusted** [comment]
Funny thing I've as well Ublock Origin with the default filter-lists and doesn't have the issue. Do you have any added/enabled filter lists on top of the default ones?

**Comment by VivekSharma1897** [comment]
That is weird because I am also using default filter-lists and everything. I am sure about it because I installed uBlock Origin for that New Profile and didn't change anything else.

**Comment by Gusted** [comment]
You are opening it on a new tab. Instead of the same tab. Now I can reproduce the issue.

**Comment by VivekSharma1897** [comment]
I did mention in the issue "Open in background tab" :P

**Comment by Gusted** [comment]
I might've missed that, it seems like it only happens on the initial load of that tab(and not with an forced reload from another tab) and thus I cannot load any debug code while the site is loading in the background. Ill give it a try a few days from now to get a fresh look at it as I don't see much to debug, but for now you can load it in the same-tab as a workaround.

Regards,
Gusted

**Comment by Shaponja** [comment]
I still have this issue. Any permanent fixes yet?

**Comment by Gusted** [comment]
Well at least we know it's happening on multiple sites.. https://github.com/darkreader/darkreader/issues/8310#issuecomment-1055852942
